yt_dlp/extractor/watchthechosen.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+import json
+
+from .common import InfoExtractor
+from ..utils.traversal import traverse_obj
```

Comment:
```suggestion
from ..utils import unified_strdate
from ..utils.traversal import traverse_obj
```
let's use this instead